latexify/core.py

```diff
@@ -146,16 +110,7 @@ def _wrap(child):
       return r'\mathrm{unknown\_uniop}(' + self.visit(node.operand) + ')'
 
   def visit_BinOp(self, node):
-    priority = {
-        ast.Add: 10,
-        ast.Sub: 10,
-        ast.Mult: 20,
-        ast.MatMult: 20,
-        ast.Div: 20,
-        ast.FloorDiv: 20,
-        ast.Mod: 20,
-        ast.Pow: 30,
-    }
+    priority = constants.BIN_OP_PRIORITY
```

Comment:
@odashi @ShigekiKarita
Same as `constants.BUILTIN_CALLEES` - Should I declare a local variable here to access it more simply in the function definition or not?

Reply:
Personally it looks reasonable to introduce shorter alias.

Reply:
I keep it as it is for now. Please let me know in case you expected an alias shorter than the current one - like `p` or `prio`.

Reply:
I meant "priority" is the shorter alias. It looks good.

Reply:
Thanks for the clarification!